src/rgw/driver/rados/rgw_rados.cc

```diff
@@ -6809,6 +6809,13 @@ int RGWRados::Object::Read::prepare(optional_yield y, const DoutPrefixProvider *
   RGWBucketInfo& bucket_info = source->get_bucket_info();
 
   if (params.part_num) {
+    map<string, bufferlist> src_attrset;
+    for (auto& iter : astate->attrset) {
+        if (boost::algorithm::starts_with(iter.first, RGW_ATTR_CRYPT_PREFIX)) {
+            ldpp_dout(dpp, 4) << "get src crypt attr: " << iter.first << dendl;
```

Comment:
please stick to 2-space indent to be consistent with https://github.com/ceph/ceph/blob/main/CodingStyle#L76-L77